lib/commands/general.js

```diff
@@ -4,12 +4,30 @@ import { util } from 'appium-support';
 import B from 'bluebird';
 import log from '../logger';
 import { NATIVE_WIN } from '../webview-helpers';
+import os from 'os';
 
 
 let commands = {}, helpers = {}, extensions = {};
 
-const logTypesSupported = {
-  'logcat' : 'Logs for Android applications on real device and emulators via ADB'
+const SUPPORTED_LOG_TYPES = {
+  logcat: {
+    description: 'Logs for Android applications on real device and emulators via ADB',
+    getter: async (self) => await self.adb.getLogcatLogs(),
+  },
+  bugreport: {
+    description: `'adb buregport' output for advanced issues diagnostic`,
+    getter: async (self) => (await self.adb.bugreport()).split(os.EOL),
+  },
+  server: {
+    description: 'Appium server logs',
+    getter: (self) => {
+      if (!self.relaxedSecurityEnabled) {
+        throw new Error('Appium server must have relaxed security flag set ' +
```

Comment:
what's the potential security hazard of seeing the appium logs?

Reply:
one can see what is happening in other parallel sessions running on the same server

Reply:
riiiight, parallel sessions. 👍